test/unit/auth/credential.spec.ts

```diff
@@ -440,6 +441,49 @@ describe('Credential', () => {
       expect((getApplicationDefault())).to.be.an.instanceof(RefreshTokenCredential);
     });
 
+    it('should return a RefreshTokenCredential with FIREBASE_EMULATOR_CREDENTIALS', () => {
+      const firebaseEmulatorCred = {
+        client_id: "fakeclientid",
+        client_secret: "fakeclientsecret",
+        refresh_token: "fakerefreshtoken",
+        type: "authorized_user",
+      };
+      delete process.env.GOOGLE_APPLICATION_CREDENTIALS;
+      process.env.FIREBASE_EMULATOR_CREDENTIALS = JSON.stringify(firebaseEmulatorCred);
+
+      const defaultCred = getApplicationDefault();
+      expect(defaultCred).to.be.an.instanceof(RefreshTokenCredential);
+      expect((defaultCred as RefreshTokenCredential).refreshToken.refreshToken).to.eql(firebaseEmulatorCred.refresh_token);
+    });
+
+    it('should throw if FIREBASE_EMULATOR_CREDENTIALS is invalid', () => {
+      process.env.FIREBASE_EMULATOR_CREDENTIALS = "{{}}";
+      expect(() => getApplicationDefault()).to.throw(Error);
+    });
+
+    it('should prefer FIREBASE_EMULATOR_CREDENTIALS to gcloud default', () => {
+      if (!fs.existsSync(GCLOUD_CREDENTIAL_PATH)) {
+        // tslint:disable-next-line:no-console
+        console.log(
+          'WARNING: Test being skipped because gcloud credentials not found. Run `gcloud beta auth ' +
+          'application-default login`.');
+        return;
+      }
+      delete process.env.GOOGLE_APPLICATION_CREDENTIALS;
+
+      const firebaseEmulatorCred = {
+        client_id: "fakeclientid",
+        client_secret: "fakeclientsecret",
+        refresh_token: "fakerefreshtoken",
+        type: "authorized_user",
+      };
+      process.env.FIREBASE_EMULATOR_CREDENTIALS = JSON.stringify(firebaseEmulatorCred);
+
+      const defaultCred = getApplicationDefault();
+      expect(defaultCred).to.be.an.instanceof(RefreshTokenCredential);
+      expect((defaultCred as RefreshTokenCredential).refreshToken.refreshToken).to.eql(firebaseEmulatorCred.refresh_token);
+    });
+
```

Comment:
Also add a test case for `isApplicationDefault()`.

Reply:
Done.  I made `implicit` false but I wasn't sure if that was the correct choice.  Does that seem right to you?